documentation/source/overview/references.rst

```diff
@@ -100,285 +100,346 @@ The table below provides references relevant to the :ref:`pam50` used by SCT, in
    * - Template/atlas
      - References
    * - PAM50 template
-     - `De Leener B, Fonov VS, Louis Collins D, Callot V, Stikov N, Cohen-Adad J. PAM50: Unbiased multimodal template of the brainstem and spinal cord aligned with the ICBM152 space. Neuroimage 2017. <http://www.sciencedirect.com/science/article/pii/S1053811917308686>`__
+     - `De Leener B, Fonov VS, Louis Collins D, Callot V, Stikov N, Cohen-Adad J.
+        PAM50: Unbiased multimodal template of the brainstem and spinal cord aligned with the ICBM152 space.
+        Neuroimage 2017.
+        <http://www.sciencedirect.com/science/article/pii/S1053811917308686>`__
```

Comment:
It looks like Sphinx isn't happy with the changes in `references.rst`. We've got 84 warnings, all for `WARNING: Inline interpreted text or phrase reference start-string without end-string.` in this file.

When I made this table, I had to wrestle with the [`list-table`](https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table) directive to get it to work. Each sub-bullet point (`-`) needs to be a single line, I believe, so the linebreaks here are wreaking havoc. :frowning_face:

Reply:
Funny thing, I thought I had seen some multi-line bullet points elsewhere in that file, hence why I had thought it wouldn't break anything. Maybe it's something to do with ` vs `` or something? Do you have some easy command I can run to check this stuff on Linux locally?

Reply:
You can `make html` in the documentation folder to run sphinx locally.

Reply:
So, I built the HTML docs (at least that part of it) locally, and it seems indeed that sphinx/rst really doesn't like line breaks in hyperlinks... I tried adding `\` at the end of the lines and that doesn't make it magically work. Is it 100% impossible to do that, and the contents really must be on a single line?

I don't mind, and I can change my commit regarding the "references.rst" file in this PR to be single line for all those hyperlinked references, but I prefer asking first before I undo all that clean formatting ;)

Reply:
i'm fine with single line.

Reply:
I could do single-line, but now I've got some curious finding: the [existing file](https://raw.githubusercontent.com/neuropoly/spinalcordtoolbox/master/documentation/source/overview/references.rst) (before any of my changes) already had line breaks, yet somehow the documentation seen [over there](https://spinalcordtoolbox.com/en/stable/overview/references.html) was rendered correctly. How did this ever work?

Reply:
> (even if you switch to vanilla master for example, you get the big pile of WARNING: Inline interpreted text or phrase reference start-string without end-string!)

I tried this locally, and I wasn't able to get any warnings. Hrm...

<details>
<summary> Long output log hidden, click to show </summary>

```bash
(venv_sct) joshua@XPS-15-9560:~/repos/spinalcordtoolbox/documentation$ make clean html
Running Sphinx v3.4.3
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 20 source files that are out of date
updating environment: [new config] 20 added, 0 changed, 0 removed
/home/joshua/repos/spinalcordtoolbox/python/envs/venv_sct/lib/python3.6/site-packages/recommonmark/parser.py:75: UserWarning: Container node skipped: type=document                                                                 
  warn("Container node skipped: type={0}".format(mdnode.t))
/home/joshua/repos/spinalcordtoolbox/python/envs/venv_sct/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
  return f(*args, **kwds)
/home/joshua/repos/spinalcordtoolbox/python/envs/venv_sct/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
  return f(*args, **kwds)
/home/joshua/repos/spinalcordtoolbox/python/envs/venv_sct/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
  return f(*args, **kwds)
reading sources... [100%] user_section/tutorials/spinalcord_segmentation                                                                                                                                                            
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] user_section/tutorials/spinalcord_segmentation                                                                                                                                                             
generating indices... genindex py-modindex done
highlighting module code... [100%] spinalcordtoolbox.vertebrae.detect_c2c3                                                                                                                                                          
writing additional pages... search done
copying images... [100%] overview/../../imgs/overview.png                                                                                                                                                                           
copying downloadable files... [100%] ../../contrib/docker/Erase_fingerprint_docker.sh                                                                                                                                               
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded.

The HTML pages are in build/html.
```